site/app/views/grading/ElectronicGraderView.php

```diff
@@ -789,7 +788,24 @@ public function hwGradingPage(Gradeable $gradeable, GradedGradeable $graded_grad
                 "message" => "Late Submission (No on time submission available)"
             ]);
         }
+        elseif ($graded_gradeable->getAutoGradedGradeable()->hasSubmission() && count($display_version_instance->getFiles()["submissions"]) > 1 && $graded_gradeable->getGradeable()->isScannedExam()) {
```

Comment:
One more issue, this appears to be flagging the other files that get uploaded with bulk uploads as extra files when they shouldn't be.
you should ignore the following files:
 ```
.submit.timestamp
bulk_upload_data.json
`